[MI-2435] Updated documetation to add the info about auto-connect feature (#44)
Brightscout/mattermost-plugin-exchange-calendar: README.md

  
 You may use the `Regenerate` button to generate a new key. Ensure that the key is configured in the mattermost-plugin-exchange-ews-proxy's `.env` file so that the proxy can authenticate API requests made by this plugin.

>   - **Auto-Connect Users**: When set to 'true', all the users on Mattermost are automatically connected to Exchange via EWS proxy server.

Reviewer: ```suggestion
  - **Auto-Connect Users**: When set to `true`, all the users on Mattermost are automatically connected to Exchange via EWS proxy server.
```

>   - **Auto-Connect Users**: When set to `true`, all the users on Mattermost are automatically connected to Exchange via EWS proxy server.

Reviewer 2: Dont think there is a need for EWS proxy server here.
```suggestion
  - **Auto-Connect Users**: When set to `true`, all the users on Mattermost will be automatically connected to Exchange Calendar.
```

>     **Note:** If an admin changes this setting from `false` -> `true`, then he/she has to disable and re-enable the plugin to auto-connect all the logged-in users. If he/she does not do so, then only the users who log in to Mattermost after the setting was enabled will auto-connect.

Reviewer: ```suggestion
    **Note:** If an admin changes this setting from `false` -> `true`, then the user has to re-enable the plugin to auto-connect all the logged-in users. If the user does not do so, then only the users who log in to Mattermost after the setting was enabled will auto-connect.
```

>     **Note:** If an admin changes this setting from `false` -> `true`, then admin has to disable and re-enable the plugin to auto-connect all the logged-in users. If the admin does not do so, then only the users who log in to Mattermost after the setting was enabled will auto-connect.

Reviewer 2: We can frame this better as just an example don't copy this.

By updating the value from false to true only the users who log in to Mattermost after the setting was enabled will auto-connect. To auto-connect all the logged-in users the admin has to disable and re-enable the plugin.
